root/etc/s6-overlay/s6-rc.d/svc-tailscale-up/run

```diff
@@ -26,18 +26,13 @@ tailscale up $FLAGS
 
 # configure serve
 if [ -v TAILSCALE_SERVE_PORT ] && [ -v TAILSCALE_SERVE_MODE ]; then
-    if [[ $TAILSCALE_SERVE_MODE == "https" ]]; then
-        tailscale serve "${TAILSCALE_SERVE_MODE}":443 / http://localhost:"${TAILSCALE_SERVE_PORT}"
-    else
-        tailscale serve "${TAILSCALE_SERVE_MODE}":443 http://localhost:"${TAILSCALE_SERVE_PORT}"
-    fi
+    tailscale serve --bg --"${TAILSCALE_SERVE_MODE}"=443 http://localhost:"${TAILSCALE_SERVE_PORT}"
 fi
 
 # configure funnel
 if [ -v TAILSCALE_FUNNEL ]; then
-    tailscale funnel 443 on
+    tailscale funnel --bg --"${TAILSCALE_SERVE_MODE}"=443 http://localhost:"${TAILSCALE_SERVE_PORT}"
```

Comment:
While this will work - it would be better to call either `server` or `funnel` depending on if `TAILSCALE_FUNNEL` is provided.